### PR TITLE
Revert macros.xml

### PR DESCRIPTION
**Pull request by neoformit**
Fixes https://github.com/galaxyproject/tools-iuc/issues/5412 by reverting to old macros.xml

### CODE REVIEW

**Comment by bernt-matthias** [comment]
So... Tests do not run (because only macro is changed).

Wondering if this is fine? I guess so. The tool that has been changed by the previous pr works anyway.. And the other ones have not been modified in the TS.

Or?

**Comment by mvdbeek** [comment]
Can you bump the version or is that also completely contained in the macro ?

**Comment by bernt-matthias** [comment]
I checked three tools: they had a hardcoded version like `1.0.1`.

**Comment by neoformit** [comment]
Hmmm... should these tools have `+galaxyN` versions? I can't bump `version="1.0.2"`?

**Comment by bernt-matthias** [comment]
Jep, just bump those versions.

**Comment by neoformit** [comment]
Hmm ok, I can do that, but isn't it misleading if the tool form says `1.0.2` when the fastx_toolkit version is actually `0.0.14`?

**Comment by mvdbeek** [comment]
You can't go backwards though, that'll mess with the latest version ordering. So the best thing to do is bump the +galaxyN suffix until the time upstream eventually catches up.

**Comment by bernt-matthias** [comment]
It is, but we can't change it :(

**Comment by neoformit** [comment]
I think we agreed on this :point_up:

**Comment by bernt-matthias** [comment]
Can you do the same for all tools of the suite? Ideally adding a `VERSION_SUFFIX` token to the macros.

**Comment by neoformit** [comment]
`fastx_renamer` already has `+galaxy1` suffix, so I have set `VERSION_SUFFIX=1` and annotated the entire suite with `version="x.x.x+galaxy@VERSION_SUFFIX@"`. Is that ok?

**Comment by neoformit** [comment]
@bernt-matthias I am not sure how to drop the `from_file` attribute that planemo lint is complaining about.
Apparently it is deprecated in favour of [from_data_table](https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-inputs-param-options) but I'm not sure if this change will break current installations? I assume `fastx_clipper_sequences.txt.sample` needs to be converted to `fastx_clipper_sequences.loc.sample` for this to work?

**Comment by bernt-matthias** [comment]
Leave it as it is.

Warnings are mainly for newly added tools. I will try to modify our workflow such that we can merge and deployment still happens.

**Comment by bernt-matthias** [comment]
Good to merge?

**Comment by neoformit** [comment]
I'm happy :+1:

File: tool_collections/fastx_toolkit/fastx_renamer/fastx_renamer.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<tool id="cshl_fastx_renamer" name="Rename sequences" version="@VERSION@+galaxy1" >
+<tool id="cshl_fastx_renamer" name="Rename sequences" version="@VERSION@+galaxy@VERSION_SUFFIX@" >
```
Comment (bgruening) [inline tool_collections/fastx_toolkit/fastx_renamer/fastx_renamer.xml:1]: this here had a version suffix of 1 and not its falling back to 0

**Comment by bgruening** [comment]
When we touch those tools maybe we can add a profile version and bump the build number to 2?

File: tool_collections/fastx_toolkit/fastx_renamer/fastx_renamer.xml (hunk 1 above)
Comment (neoformit) [reply tool_collections/fastx_toolkit/fastx_renamer/fastx_renamer.xml:1]: Good catch, I meant to set the version suffix to 1, not 0.

**Comment by neoformit** [comment]
Shall I just add a profile version to them now?

**Comment by bgruening** [comment]
I would say so. And bump to version 2 so all of them get a update.

**Comment by neoformit** [comment]
Done

**Comment by bernt-matthias** [comment]
Thanks @neoformit

**Pull request merged**